doc/en/user/source/rest/api/index.rst

```diff
@@ -3,7 +3,7 @@
 REST configuration API reference
 ================================
 
-This section describes the GeoServer REST configuration API.
+This section describes the GeoServer REST configuration API.  You may also want to refer to the `Swagger spec <https://docs.geoserver.org/latest/en/user/rest/index.html#api>`_ but heed the warning.
```

Comment:
What does this provided? reading ...

Better to repeat the what is there to set expectations ...